deploy/docker-compose/docker-compose.yaml

```
@@ -0,0 +1,41 @@
+version: '3'
+
+services:
+  cache:
+    depends_on:
+      - tfserving
+    image: tfservingcache
+    build: 
+        context: ./../..
+        dockerfile: build/docker/Dockerfile
```

Looks like the dockerfile path should be:
```suggestion
        dockerfile: deploy/docker/Dockerfile
```